doc/tutorial/conv_arithmetic.txt

```diff
@@ -0,0 +1,180 @@
+.. _conv_arithmetic:
+
+======================
+Convolution arithmetic
+======================
+
+The relationship between a convolution operation's input shape, kernel size,
+stride, padding and its output shape can be confusing at times.
+
+This tutorial aims at providing an intuitive understanding of how the different
+convolution parameters influence the shape of the output it produces.
+
+Refresher: discrete convolution
+===============================
+
+.. todo::
+
+    WRITEME
+
+Here are the factors determining the shape of a convolution's output size along
+a given axis (height or width):
+
+* **Input size**: length of the input axis.
+* **Kernel size**: length of the kernel.
+* **Stride**: distance between two consecutive positions of the kernel.
+* **(Zero-) padding size**: number of zero values concatenated at the beginning
+  and at the end of the axis.
+
+For the remainder of this tutorial, we will concentrate on convolutions mapping
+one input feature map to one output feature map. The relationships presented
+here also hold in the general case.
```

Comment:
I'd specify we will consider 2D convolutions. Also, I would state explicitly what *the general case* is.